### PR TITLE
Interacting with the DOM Lesson

### PR DESCRIPTION
**Pull request by bfwetmore**
- Add all rough work associated with interacting with the DOM lesson.
- Add practice file on traversing the DOM.

### CODE REVIEW

File: interacting-dom/app.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+const btnCreate = document.getElementById("btn-main");
+const btnToggle = document.querySelector(".btn-toggle");
+const btnRemove = document.querySelector(".btn-remove");
+const taskList = document.querySelector(".list-container ul");
+const remove = document.querySelectorAll('li');
+for (let i = 0; i < remove.length; i++) {
+    attachRemoveButton(remove[i]);
+}
+
+function attachRemoveButton(poop) {
```
Comment (do-work) [inline interacting-dom/app.js]: Description parameter names are preferred.

File: interacting-dom/app.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    let remove = document.createElement('button');
+    remove.className = 'remove';
+    remove.textContent = 'Remove';
+    poop.appendChild(remove);
+}
+
+// const listItem = document.getElementsByTagName("li");
+
+// // for (let i = 0; i < listItem.length; i += 1) {
```
Comment (do-work) [inline interacting-dom/app.js]: Scan the code for commented out code and remove those before doing a PR.

File: interacting-dom/app.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    const listContainer = document.querySelector(".list-container");
+    if (listContainer.style.display === "none") {
+        btnToggle.textContent = "Hide List";
+        listContainer.removeAttribute("style");
+    } else {
+        btnToggle.textContent = "Show List";
+        listContainer.style.display = "none";
+    }
+});
+btnCreate.style.backgroundColor;
```
Comment (do-work) [inline interacting-dom/app.js]: Is this line needed?

**Review by do-work — CHANGES_REQUESTED** [review]

File: interacting-dom/app.js (hunk 1 above)
Comment (bfwetmore) [reply interacting-dom/app.js]: Fixed

File: interacting-dom/app.js (hunk 2 above)
Comment (bfwetmore) [reply interacting-dom/app.js]: Done

File: interacting-dom/app.js (hunk 3 above)
Comment (bfwetmore) [reply interacting-dom/app.js]: No, deleted

**Review by do-work — APPROVED** [review]

**Pull request merged**